### DISCUSSION TITLE
Add speedometer chart controlled by analog servo

### DISCUSSION THREAD

**Discussion by hag2001** [tbnobody/OpenDTU#791] [Show and tell]
### Describe the solution you'd like

Demo: Speedometer controlled by Analog Servo, 4.8-6.0V, 0.12/0.1 s/60° 

![2023-04-10 13 19 21](https://user-images.githubusercontent.com/33687528/230896181-077ba249-68eb-4b94-9f28-771923434a4c.jpg)
Servo:
![2023-04-10 12 59 28](https://user-images.githubusercontent.com/33687528/230896498-72914470-6e2f-483e-9e91-4ba76dec3fd3.jpg)

**Comment by tbnobody** [comment]
This Projekt is meant as a  bridge between the hoymiles protocol and mqtt / web UI. Therefor I will not implement such this as servo control etc. You are free to realize this by modifying the code yourself or use a second ESP which connects to opendtu using web UI or mqtt. 
But unfortunately, in order to keep the code clean, stable and expandable, i can't include any display idea.

**Comment by hag2001** [comment]
Sure, I've implemented this already ... If somebody is interested ...

**Comment by tbnobody** [comment]
I am going to move this to the discussions as it is more visibile there
